templates/base.html

```diff
@@ -93,7 +93,7 @@
               nav-item
             "
           >
-            <a class="nav-link" href="{% url 'membership-secretary' %}">Directory</a>
+            <a class="nav-link" href="{% url 'membership-secretary' %}">Board Directory</a>
```

Comment:
Is this change for the membership directory?

Reply:
Yes  the name is changing to distinguish between the two.